invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -365,20 +367,169 @@ def view(**kwargs):
     return view
 
 
-def add_signposting(f):
-    """Add signposting link to view's response headers."""
+def _get_header(rel, value, link_type=None):
+    header = f'<{value}> ; rel="{rel}"'
+    if link_type:
+        header += f' ; type="{link_type}"'
+    return header
+
+
+def _get_signposting_cite_as(record):
+    """Release self url points to RDM record.
+
+    It points to DataCite URL if the integration is enabled, otherwise it points to the HTML URL.
+    """
+    doi_url = record["links"].get("doi")
+    html_url = record["links"]["self_html"]
+    return _get_header("cite-as", doi_url or html_url)
+
+
+def _get_signposting_types(record):
+    resource_type = record["metadata"]["resource_type"]
+    props = get_vocabulary_props(
+        "resourcetypes",
+        [
+            "props.schema.org",
+        ],
+        resource_type["id"],
+    )
+    url_schema_org = props.get("schema.org")
+    return [
+        _get_header("type", url_schema_org),
+        _get_header("type", "https://schema.org/AboutPage"),
+    ]
+
+
+def _get_signposting_authors(record):
+    authors = []
+    # Limit authors to the first 10.
+    for creator in islice(record["metadata"]["creators"], 0, 10):
+        for identifier in creator["person_or_org"].get("identifiers", []):
+            if identifier["scheme"] == "orcid":
+                authors.append(
+                    _get_header(
+                        "author", "https://orcid.org/" + identifier["identifier"]
+                    )
+                )
+    return authors
+
+
+def _get_signposting_describedbys(pid_value):
+    describedbys = []
+    for export_format, val in current_app.config.get(
+        "APP_RDM_RECORD_EXPORTERS", {}
+    ).items():
+        url = export_url_for(pid_value=pid_value, export_format=export_format)
+        content_type = val["content-type"]
+        describedbys.append(_get_header("describedby", url, content_type))
+    return describedbys
+
+
+def _get_signposting_licenses(record):
+    licenses = []
+    for right in record["metadata"].get("rights", []):
+        # First try to get `props.url` from the standard licenses,
+        # then try to get the optional `link` from the custom license.
+        url = right.get("props", {}).get("url") or right.get("link")
+        if url:
+            licenses.append(_get_header("license", url))
```

Comment:
The FAIR Signposting docs recommends to use SPDX license identifier (e.g. `https://spdx.org/licenses/CC0-1.0`).
However, [in Zenodo we store URLs like `https://creativecommons.org/publicdomain/zero/1.0/legalcode`](https://github.com/zenodo/zenodo-rdm/blob/a11ff414c1a3c3829c26868a38aae3110b3fd25e/app_data/vocabularies/licenses.csv#L101) and not `spdx.org` URLs.

Reply:
If `props["scheme"] == "spdx"` I think we can safely generate the URL like `https://spdx.org/licenses/{right["id"]}`. We might have licenses (or even non-SPDX licenses), in which case just using `url` like here would be ok.

Reply:
Ouch, I tried in the browser and copy-pasting URLs for some reason kept the original case... Ok, this is a bummer, I think we'll have to add the original `spdx` ID with the exact case as a `props.spdx_id` field or similar... 

I think it would be fine to shelve this and just use the `url`, depends on whether we want to spend more time to re-import SPDX and update the existing license vocabulary (funnily, the dump we have is from more than a year ago).